iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/compaction/execute/utils/CompactionUtils.java

```diff
@@ -189,20 +200,17 @@ public static void addFilesToFileMetrics(TsFileResource resource) {
             resource.getTsFile().length(),
             resource.isSeq(),
             resource.getTsFile().getName());
-    if (resource.modFileExists()) {
+    if (resource.anyModFileExists()) {
```

Comment:
For the shared mods file, is this redundant addition here?

Reply:
Done